cv-data-service-library/src/main/java/com/trihydro/library/helpers/TimGenerationHelper.java

```diff
@@ -149,26 +154,39 @@ public List<ResubmitTimException> updateAndResubmitToOde(List<ActiveTimValidatio
                 }
 
                 WydotTim wydotTim = getWydotTimFromTum(tum);
-                List<Milepost> mps = new ArrayList<>();
+                List<Milepost> mps;
                 List<Milepost> allMps = getAllMps(wydotTim);
                 if (allMps.size() < 2) {
                     String exMsg = String.format(
-                            "Unable to resubmit TIM, less than 2 mileposts found to determine service area for Active_Tim %d",
-                            tum.getActiveTimId());
+                        "Unable to resubmit TIM, less than 2 mileposts found to determine service area for Active_Tim %d",
+                        tum.getActiveTimId());
                     utility.logWithDate(exMsg);
                     exceptions.add(new ResubmitTimException(activeTimId, exMsg));
                     continue;
                 }
+
                 Milepost firstPoint = allMps.get(0);
                 Milepost secondPoint = allMps.get(1);
 
+                Milepost anchorMp;
+                try {
+                    anchorMp = getAnchorPoint(firstPoint, secondPoint);
+                } catch (Utility.IdenticalPointsException e) {
+                    String exMsg = String.format(
+                        "Unable to resubmit TIM, identical points found while calculating anchor point for Active_Tim %d",
+                        tum.getActiveTimId());
+                    utility.logWithDate(exMsg);
+                    exceptions.add(new ResubmitTimException(activeTimId, exMsg));
+                    continue;
+                }
```

Comment:
Clarification (non-blocking): This is an instance of the new exception being caught by the TimGenerationHelper. This is now the behavior wherever `getAnchorPoint` is called.